docs/HowTo/Get-Started/Manage-Memory.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Teku memory management
+---
+
+# Managing memory
+
+Manage Teku's Java Virtual Machine (JVM) memory requirements by setting a maximum heap size
+using the `JAVA_OPTS` environment variable.
+
+We recommend setting the maximum heap size to 2GB if you have less than 8GB of RAM. This results in
```

Comment:
Might be worth noting that setting the max heap is also useful if you have a lot of RAM and want Teku to use less of it to leave more room for other things.

Reply:
Done